### PR TITLE
BO-38 Module for goreplay

### PR DESCRIPTION
**Pull request by ppp0**

### CODE REVIEW

**Comment by ppp0** [comment]
@kris-lab please review

**Comment by ppp0** [comment]
@kris-lab I'm going to self-merge this, fyi - we can always undo this if you do not agree

File: modules/goreplay/manifests/init.pp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class goreplay( $version = '0.15.1') {
```
Comment (kris-lab) [inline modules/goreplay/manifests/init.pp]: formatting

File: modules/goreplay/manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+  helper::script { 'install goreplay':
+    content => template("${module_name}/install.sh.erb"),
+    unless  => 'which gor',
```
Comment (kris-lab) [inline modules/goreplay/manifests/init.pp]: Check if binary exists at requested path `/usr/local/bin/gor` with requested version?

```
$ gor
Version: v0.15.1
2017/03/07 17:35:06 Required at least 1 input and 1 output
```

File: modules/goreplay/manifests/init.pp [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  }
+}
```
Comment (kris-lab) [inline modules/goreplay/manifests/init.pp]: new line

**Pull request merged**